includes/results.inc

```diff
@@ -863,10 +864,11 @@ class IslandoraSolrFacets {
   /**
    * Process facet ranges.
    *
-   * Not in place yet.
+   * Based on the existing implementation for date fields, but modified in place
+   * to handle differences between date and range fields.
    */
   public function processFacetRanges() {
-
+    $this->processFacetDates();
```

Comment:
I'm not entirely sure this is appropriate, as range fields can cover things that are not dates... integers or other numerical data... but here, we do some pretty explicit [date formatting](https://github.com/Islandora/islandora_solr_search/pull/342/files#diff-17a5945b1aa412dec3b39453ad29425fR940).

Reply:
I'm not disagreeing with what you are saying here. Just that some range formatting seems better than none. Once we have someone using this with Solr 6+ it will be easier to see the holes. One suggestion would be to make a field a `facet_date` if its a `facet_range` but on a `islandora_solr_is_date_field()` field.